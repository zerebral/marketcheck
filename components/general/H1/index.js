import styled from 'styled-components'
import { rem, em } from 'polished'
import { mediaMax } from '%/styles/mixins'

const H1 = styled.h1`
  font-size: ${rem(48)};
  font-weight: 700;
  line-height: ${58 / 48};
  margin-bottom: ${em(20, 48)};
  ${mediaMax.desktop`font-size: ${rem(40)};`}
  ${mediaMax.laptop`font-size: ${rem(36)};`}
  ${mediaMax.tablet`font-size: ${rem(30)};`}
  ${mediaMax.phablet`font-size: ${rem(26)};`}
`

export default H1
