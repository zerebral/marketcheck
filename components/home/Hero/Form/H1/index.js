import styled from 'styled-components'
import { rem } from 'polished'
import { colors } from '%/styles'

const H1 = styled.h1`
  text-align: center;
  color: ${colors.white};
  font-size: ${rem(40)};
  font-weight: 700;
`

export default H1
