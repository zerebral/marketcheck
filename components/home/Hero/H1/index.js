import styled from 'styled-components'
import { rem, em } from 'polished'
import { colors } from '%/styles'
import { mediaMax } from '%/styles/mixins'

const H1 = styled.h1`
  text-align: center;
  color: ${colors.white};
  font-size: ${rem(40)};
  font-weight: 700;
  margin-bottom: ${em(20, 40)};

  ${mediaMax.laptop`
    font-size: ${rem(30)};
  `}

  ${mediaMax.tablet`
    font-size: ${rem(26)};
    max-width: 80%;
    margin-left: auto;
    margin-right: auto;
  `}
`

export default H1
