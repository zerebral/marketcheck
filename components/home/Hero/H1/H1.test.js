/* eslint-env jest */
import React from 'react'
import { shallow } from 'enzyme'
import 'jest-styled-components'

import H1 from './index'

describe('H1 Component', () => {
  test('H1 Passes shallow snapshot', () => {
    const wrapper = shallow(<H1>Hello World</H1>)
    expect(wrapper).toMatchSnapshot()
  })
})
