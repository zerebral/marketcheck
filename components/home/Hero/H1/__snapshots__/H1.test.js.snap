// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`H1 Component H1 Passes shallow snapshot 1`] = `
.c0 {
  text-align: center;
  color: #ffffff;
  font-size: 2.5rem;
  font-weight: 700;
  margin-bottom: 0.5em;
}

<h1
  className="c0"
>
  Hello World
</h1>
`;
